modules/sdk-coin-cspr/tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "outDir": "./dist",
+    "rootDir": "./",
+    "strictPropertyInitialization": false,
+    "esModuleInterop": true,
+    "typeRoots": ["../../types", "./node_modules/@types", "../../node_modules/@types"]
+  },
+  "include": ["src/**/*"],
```

Comment:
can you add `test/**/*` to the include here, & then make sure to exclude them in the `.npmignore` file:
```
dist/test/
dist/tsconfig.tsbuildinfo